src/index.html

```diff
@@ -9,10 +9,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The stylesheet is linked as an SCSS file (`index.scss`). Browsers cannot directly interpret SCSS files. You need to compile the SCSS file to a CSS file and link the resulting CSS file instead.